frontend/src/components/registration-form/components/user-form/user.form.module.scss

```diff
@@ -0,0 +1,41 @@
+@import "styles/main.scss";

```

Comment:
Ovo mi izgleda kao pogresno resen konflikt... 
Ovaj fajl ne bi trebao da postoji. 
Sve vezano za registracionu formu bi trebalo da bude u `frontend/src/components/registration`. 
Mislim da je pre to sve bilo u `frontend/src/components/registration-form`, ali sad je taj folder obrisan.
@nemanjaporobic ako mozes samo ovo da potvrdis

Reply:
@milos1302 , @M1K1B , tako je, registration-form folder mozes obrisati skroz, iz tog foldera je sve prebaceno u registration folder

Reply:
Vazi, ispravicu to